.github/renovate.json5

```
@@ -136,9 +136,9 @@
         "go",
         "golang"
       ],
-      "allowedVersions": "<=1.19",
+      "allowedVersions": "<=1.20",
```

Would the following work?

```suggestion
      "matchUpdateTypes": ["patch", "digest", "pin", "pinDigest"],
```

If yes then we wouldn't have to track the Go version every time we branch to a new minor version.